pom.xml

```diff
@@ -258,6 +258,16 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.jenkins-ci.tools</groupId>
+        <artifactId>maven-hpi-plugin</artifactId>
+        <configuration>
+          <minimumJavaVersion>1.8</minimumJavaVersion>
+          <maskClasses>
```

Comment:
Is this needed?

> ws|tab|CR|LF]+ separated list of package prefixes that your plugin doesn't want to see from the core.
Tokens in this list is prefix-matched against the fully-qualified class name, so add "." to the end of each package name, like "com.foo. com.bar."

Reply:
Yes, in this PR, it will create objects like `SecretStringCredentials` which also depend on the Keyvault SDK. If shared classes are not masked here, it may cause some exceptions like ClassNotFound.